_getting-started/search-data.md

```diff
@@ -0,0 +1,444 @@
+
+```json
+GET /students/_search
+{
+  "query": {
+    "match": {
+      "name.keyword": "john"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+This request returns no hits because the `keyword` fields must be matched exactly. 
+
+However, you can search for the exact text `John Doe`:
+
+```json
+GET /students/_search
+{
+  "query": {
+    "match": {
+      "name.keyword": "John Doe"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+OpenSearch returns the matching document:
+
+```json
+{
+  "took": 19,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 0.9808291,
+    "hits": [
+      {
+        "_index": "students",
+        "_id": "1",
+        "_score": 0.9808291,
+        "_source": {
+          "name": "John Doe",
+          "gpa": 3.89,
+          "grad_year": 2022
+        }
+      }
+    ]
+  }
+}
+```
+
+### Filters
+
+You can add a filter clause to your query for fields with exact values using a Boolean query. 
+
+Term filters match specific terms. For example, the following Boolean query searches for students whose graduation year is 2022:
+
+```json
+GET students/_search
+{
+  "query": { 
+    "bool": { 
+      "filter": [ 
+        { "term":  { "grad_year": 2022 }}
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Range filters support specifying a range of values. For example, the following Boolean query searches for students whose GPA is greater than 3.6:
+
+```json
+GET students/_search
+{
+  "query": { 
+    "bool": { 
+      "filter": [ 
+        { "range": { "gpa": { "gt": 3.6 }}}
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+For more information about filters, see [Query and filter context]({{site.url}}{{site.baseurl}}/query-dsl/query-filter-context/).
+
+### Compound queries
+
+A compound query lets you combine multiple query or filter clauses. A Boolean query is an example of a compound query.
+
+For example, to search for students whose name matches `doe` and filter by graduation year and GPA, use the following request:
+
+```json
+GET students/_search
+{
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "match": {
+            "name": "doe"
+          }
+        },
+        { "range": { "gpa": { "gte": 3.6, "lte": 3.9 } } },
+        { "term":  { "grad_year": 2022 }}
+      ]
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Search methods
+
+Along with the traditional BM25 search described in this tutorial, OpenSearch supports a range of machine learning (ML)-powered search methods, including k-NN, semantic, multimodal, sparse, hybrid, and conversational search. For information about all search methods, see [Search]({{site.url}}{{site.baseurl}}/search-plugins/).
```

Comment:
It looks like the latest commit removed the description of BM25. (I think this is the only mention of BM25 in the page now.)

Maybe "Along with the traditional full-text search described..." ?